### PR TITLE
[Improvement-14318][UI] migrate version 2.x workflow definition d3 tree view to version 3.x

### LINKED ISSUE #14318: [Improvement][UI] migrate version 2.x workflow definition d3 tree view to version 3.x
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

1.  When rendering the tree view component using echarts on workflow-definition page in version 3.x , I noticed that the task state of the tree nodes is only based on the first instance, which may not be accurate.

2. The echarts tree conceals other instance information in version 3.x . I think the approach used in version 2.x, is more intuitive.

3. I want to migrate  version 2.x  d3 tree view and replace version 3.x.

version 2.x:
截屏2023-04-18 14 31 43

version 3.x:
截屏2023-06-09 10 11 13

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by yeahhhz**
## Purpose of the pull request

close #14318 

## Brief change log

- migrate version 2.x d3 tree view and replace version 3.x.

## Verify this pull request

截屏2023-06-20 15 12 44

### CODE REVIEW

File: dolphinscheduler-ui/src/views/projects/workflow/definition/tree/index.tsx [hunk 1]
```diff
@@ -154,72 +188,100 @@ export default defineComponent({
...
     }
 
-    const initChartData = (node: any, newNode: any) => {
-      newNode.children = []
-      node?.children.map((child: any) => {
-        const newChild = {}
-        initChartData(child, newChild)
-        newNode.children.push(newChild)
-      })
+    // const initChartData = (node: any, newNode: any) => {
```
Comment (SbloodyS) [inline dolphinscheduler-ui/src/views/projects/workflow/definition/tree/index.tsx]: Please remove unnessnary code.

Comment (yeahhhz) [reply dolphinscheduler-ui/src/views/projects/workflow/definition/tree/index.tsx]: done

**Comment by zhongjiajie** [comment]
do you have time to take a look at this pr @devosend

**Review by songjianet — DISMISSED** [review]

**Comment by songjianet** [comment]
Please deal with the problem of front-end CI Build error. @yeahhhz

**Review by songjianet — APPROVED** [review]

**Pull request merged**